Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java

```diff
@@ -911,6 +928,80 @@ private void chooseController()
     builder.show();
   }
 
+  // TODO: Make this setting work with custom GameSettings and update setting description.
+  public static void handleWiiOverlayDisablesControllersSetting(int selection, boolean IsFirstRun)
+  {
+    File dolphinFile = SettingsFile.getSettingsFile(Settings.FILE_DOLPHIN);
+    File wiimoteNewFile = SettingsFile.getSettingsFile(Settings.FILE_WIIMOTE);
+    IniFile dolphinIni = new IniFile(dolphinFile);
+    IniFile wiimoteNewIni = new IniFile(wiimoteNewFile);
+
+    boolean wiiOverlayDisablesControllers = dolphinIni.getBoolean(Settings.SECTION_INI_ANDROID,
+            SettingsFile.KEY_WII_OVERLAY_DISABLES_CONTROLLERS, false);
```

Comment:
This is a `static` method so is the change I made acceptable?

Reply:
It'll do for now. For what it's worth, the first commit of PR #9146 should make it easy to fix this cleanly. Let me know if you want me to split out that commit to a separate PR.

Also, since you've made this change, I think `MAIN_WII_OVERLAY_DISABLES_CONTROLLERS` works properly with per-game settings now? I'm not entirely sure, but it's worth testing.

Reply:
`MAIN_WII_OVERLAY_DISABLES_CONTROLLERS` itself works with custom game settings but its usage does not yet. `EmulationActivity.handleWiiOverlayDisablesControllersSetting` doesn't modify custom game settings yet. That's a rather complex changed and it'd also be good to isolate Wii Overlay changes when using custom game settings, which should be done in another PR.